pages/dkp/kommander/2.2/dkp-upgrade/index.md

```diff
@@ -9,16 +9,56 @@ beta: false
 
 The DKP upgrade represents an important step of your environment's lifecycle, as it ensures that you are up-to-date with the latest features and can benefit from the most recent improvements, enhanced cluster management, and better performance. This section describes how to upgrade your networked, air-gapped, or on-prem environment to the latest version of DKP.
 
-## Upgrade paths
-
-Verified upgrade paths are as follows:
-
-| To    | From | 1.8.4 | 1.8.5 | 2.1.0 | 2.1.1 |
-|-------|------|:-----:|:-----:|:-----:|:-----:|
-| 1.8.5 |      | Yes   |       |       |       |
-| 2.1.0 |      | Yes   | Yes   |       |       |
-| 2.1.1 |      | Yes   | Yes   | No    |       |
-| 2.2.0 |      | No    | No    | Yes   | Yes   |
+## Supported upgrade paths
+
+|**Display Icon** | **Service** |
+|---------- | ------- |
+| ⚫ | Supported |
+| ◯ | Not Supported |
+
+<table style="border-collapse: collapse;" Border = "1" Cellpadding = "5" Cellspacing = "5">
```

Comment:
I think we need to abandon the dots due to them not being accessible.  Personally, I know that we try to keep html to a minimum, but as this table gets larger it's going to be harder to read if it's not well formatted and I think the formatting of the html is much better.

Reply:
Can we maybe just change the N/A to blank, to try and get a bit less visual clutter out?   
I *really* liked the dots because of the visual distinction they provided but understand the accessibility issue.

Reply:
yeah for sure, removing the N/A definitely would help. done in https://github.com/mesosphere/dcos-docs-site/pull/4308/commits/aea24b9a30390770a6e5f582ff48afbc2d12dc19

Reply:
no NA, I like the version with version numbers in each cell. Ala:
https://github.com/mesosphere/dcos-docs-site/pull/4308/commits/369b27a528eb12986ca3b7aa4f1886253b8dbc92?short_path=e4cd10f#diff-e4cd10fee5e0a34cdcdea407029393a440d517ce5ec4a88a536b7853f08c845e 

@mmcelroy91 can you please put this version of the table back in and then we can merge.

Reply:
thanks @cmastr @mmcelroy91, happy to get this one merge! 😅